### PR TITLE
Add "Weapon ID"

### PR DESCRIPTION
**Pull request by xpdota**
For lack of a better name, I'm calling it "Weapon ID". This seems to be the one that is applicable to Omega weapons, but I want to leave the existing TransformationID field alone since it seems to convey something that could be useful in the future (more of a full-model transformation rather than just the weapon).

### CODE REVIEW

**Review by valarnin — APPROVED** [review]
Looks good to me in general. I'm still not convinced that this is only `weapon`, e.g. that it might also involve other "equipment" changes or other attachments to the model. I just lack enough different cases to suggest checking against to be sure on the naming.

**Comment by quisquous** [comment]
Yeah agreed on the naming ambiguity, but we already have a `TransformationId`, so unclear what else this should be.  I think in the future if it becomes ambiguous, we can add a comment if it's minor, and consider some renaming/backwards compat/deprecation if it's a major issue.

(Should this get PR'd to struct traits as well?)

**Review by quisquous — APPROVED** [review]

**Comment by quisquous** [comment]
@xpdota can you PR a version bump and I'll hit the merge button? https://github.com/OverlayPlugin/OverlayPlugin/blob/main/docs/how_to_release.md#overview

(In general, feel free to propose a release by sending a version bump PR if something gets merged and you want it out.)

**Comment by xpdota** [comment]
Done https://github.com/OverlayPlugin/OverlayPlugin/pull/169

File: OverlayPlugin.Core/MemoryProcessors/Combatant/CombatantMemory61.cs [hunk 1]
```diff
@@ -214,6 +215,9 @@ private unsafe struct CombatantMemory
             [FieldOffset(0x1AB2)]
             public ushort WorldID;
 
+            [FieldOffset(0x1AC6)]
+            public byte WeaponId;
```
Comment (quisquous) [inline OverlayPlugin.Core/MemoryProcessors/Combatant/CombatantMemory61.cs:219]: Belatedly, inconsistent "Id" capitalization here with the rest of the file.  Not sure what to do about that.  Maybe we can send both for a little bit?

**Pull request merged**